src/coreComponents/constitutive/fluid/multifluid/compositional/functions/CubicEOSPhaseModel.hpp

```diff
@@ -44,6 +48,8 @@ struct PengRobinsonEOS
       ? 0.37464 + 1.54226 * omega - 0.26992 * omega * omega
       : 0.3796 + 1.485 * omega - 0.164423 * omega * omega + 0.016666 * omega * omega * omega;
   }
+
+  static constexpr char const * catalogName(){ return "PengRobinson"; }
```

Comment:
I am not sure one needs `catalogName` for those things, could you help me to understand it here?

Reply:
There was a point to this. I believe I have a coupled of commits missing. Let me check my local branch.

Reply:
Added one remaining commit. The idea is the negative two phase flash is no longer tied to the cubic eos. So the names are moved into the cubic eos objects themselves. These are used down the line in CompositionalMultiphaseFluid to give names to catalog entries.